src/pages/my_page/MyPagePage.tsx

```diff
@@ -22,6 +24,8 @@ import {
 } from '../../components/PageWithSideMenu';
 import { ErrorBoundary } from '../../components/ErrorBoundary';
 import ResearchProfile from '../research_profile/ResearchProfile';
+import { Box } from '@mui/system';
+import { Divider } from '@mui/material';
```

Comment:
Har fått samme feil import tidligere også

```suggestion
import { Box, Divider } from '@mui/material';
```